recipes/pvga/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "pvga" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e8b60edbb83a9e38831a813c4e75bdcfd4fd8c1f35b804191f24ba49e83cbd9e
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - pvga=pvga:main
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+  run_exports:
+    - {{ pin_subpackage('pvga', exact=True) }} 
+
+requirements:
+  channels:
```

```suggestion
  channels:
```
Please remove the channels item. It is not used